prac_09/unreliable_car.py

```diff
@@ -0,0 +1,23 @@
+"""Unreliable Car class"""
+
+from car import Car
+from random import randint
+
+
+class UnreliableCar(Car):
+    """Unreliable Car class"""
+
+    def __init__(self, name, fuel, reliability):
+        """Initialise UnreliableCar object"""
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def __str__(self):
+        """Return a string suited to the UnreliableCar object"""
+        return f"{super().__str__()} Reliability: {self.reliability}"
+
+    def drive(self, distance):
+        """Drive an UnreliableCar"""
+        if randint(0, 100) >= self.reliability:
+            distance = 0
+        return super().drive(distance)
```

Comment:
this appears to be doing more work than is needed, do we need to attempt to drive the car with zero distance? Wouldn't take much time in reality but it might not be the most efficient way when we know it won't drive with zero distance input.

Reply:
I believe the prac still asks us ti attempt to drive the car even if the distance is ultimately 0